test-files/cmd/glacial/serial/ServiceConflateDiffTest.sh.off

```diff
@@ -8,7 +8,8 @@ TEST_NAME=ServiceConflateDiffTest
 IN_DIR=/home/vagrant/hoot/tmp/3851
 OUT_DIR=$IN_DIR
 
-CONFIG="--warn -C Testing.conf -D writer.include.debug.tags=true -D uuid.helper.repeatable=true -D debug.maps.write=true -D api.db.email=OsmApiDbHootApiDbConflate@hoottestcpp.org -D hootapi.db.writer.create.user=true -D hootapi.db.writer.overwrite.map=true -D changeset.user.id=1"
+CONFIG="--info -C Testing.conf -D writer.include.debug.tags=true -D uuid.helper.repeatable=true -D debug.maps.write=false -D api.db.email=OsmApiDbHootApiDbConflate@hoottestcpp.org -D hootapi.db.writer.create.user=true -D hootapi.db.writer.overwrite.map=true -D changeset.user.id=1"
```

Comment:
This is just a deactivated test harness for conflate and diff experimentation. Eventually, I make an actual test out of it with a smaller dataset than Milford.